addOns/postman/src/main/java/org/zaproxy/addon/postman/PostmanParser.java

```diff
@@ -147,13 +170,23 @@ private static boolean isContentTypeAlreadySet(List<KeyValueData> headers) {
     }
 
     static HttpMessage extractHttpMessage(Item item) {
+        return extractHttpMessage(item, new ArrayList<>());
+    }
+
+    static HttpMessage extractHttpMessage(Item item, List<String> errors) {
         Request request = item.getRequest();
         if (request == null) {
+            errors.add(
+                    Constant.messages.getString(
+                            IMPORT_FORMAT_ERROR, item.getName(), "request is not present"));
             return null;
         }
 
         Url url = request.getUrl();
         if (url == null) {
+            errors.add(
+                    Constant.messages.getString(
+                            IMPORT_FORMAT_ERROR, item.getName(), "url is not present"));
```

Comment:
Note that the latter part of the error messages (e.g. `url is not present`) also needs to be in the `Messages.properties` file.